resolvers/patient.js

```diff
@@ -44,6 +66,10 @@ const patientResolvers = {
       });
     },
     updatePatient: async (parent, args) => {
+      validators.validateRole(
+        [Roles.COMMANDER, Roles.SUPERVISOR],
```

Comment:
we'll need dispatch here since they'll be able to edit run numbers - limiting their ability to only edit that field will be done in the front end
```suggestion
        [Roles.COMMANDER, Roles.SUPERVISOR, Roles.DISPATCH],
```